### PR TITLE
Account status

### PR DESCRIPTION
**Pull request by yosriz**
new API for getting current account status on blockchain - not created/not activated/ activated.
tests + sample app were added

### CODE REVIEW

File: kin-core/src/test/java/kin/core/AccountInfoRetrieverTest.java [hunk 1]
```diff
@@ -86,7 +103,15 @@ public void getBalance_SameIssuerDifferentAsset_AccountNotActivatedException() t
     }
 
     @Test
-    public void getBalance_IOException() throws Exception {
+    public void getStatus_SameIssuerDifferentAsset_StatusNotActivated() throws Exception {
```
Comment (berryve) [inline kin-core/src/test/java/kin/core/AccountInfoRetrieverTest.java:107]: what is the different between this test and getStatus_NoKinTrust_StatusNotActivated

File: sample/src/main/res/layout/wallet_activity.xml [hunk 2]
```diff
@@ -48,43 +48,83 @@
...
                 android:orientation="vertical">
-
                 <LinearLayout
                     android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:orientation="horizontal">
+                    android:layout_height="match_parent"
+                    android:orientation="vertical">
+
+                    <LinearLayout
```
Comment (berryve) [inline sample/src/main/res/layout/wallet_activity.xml:63]: I know this is a sample app but nesting of 3 linear layouts? 
consider changing to constraint layout

File: kin-core/src/test/java/kin/core/AccountInfoRetrieverTest.java (hunk 1 above)
Comment (yosriz) [reply kin-core/src/test/java/kin/core/AccountInfoRetrieverTest.java:107]: all these tests are similar to get balance tests, this one purpose is to verify i'm checking both issuer and asset code when deciding whether this account is activated (has trust line), in NoKinTrust I just don't have any trust in the account i'm tesing

File: sample/src/main/res/layout/wallet_activity.xml (hunk 2 above)
Comment (yosriz) [reply sample/src/main/res/layout/wallet_activity.xml:63]: you're right, but I really don't see a point investing effort in this, UI performance is not relevant, nor it has a great impact as we don't have animations or dynamic data here (list)

File: kin-core/src/test/java/kin/core/AccountInfoRetrieverTest.java (hunk 1 above)
Comment (berryve) [reply kin-core/src/test/java/kin/core/AccountInfoRetrieverTest.java:107]: @yosriz the tests are identical and they also use the same "balance_res_no_kin_trust.json" file

File: sample/src/main/res/layout/wallet_activity.xml (hunk 2 above)
Comment (berryve) [reply sample/src/main/res/layout/wallet_activity.xml:63]: your call but 
1. how much effort would it take?
2. this is open source code - do you not want to give a good example ?

**Review by berryve — APPROVED** [review]

File: kin-core/src/test/java/kin/core/AccountInfoRetrieverTest.java (hunk 1 above)
Comment (yosriz) [reply kin-core/src/test/java/kin/core/AccountInfoRetrieverTest.java:107]: oh... now I see. I need to use a different json here 'balance_res_same_issuer_different_asset_code.json', the same goes for `getBalance_SameIssuerDifferentAsset_AccountNotActivatedException`

File: sample/src/main/res/layout/wallet_activity.xml (hunk 2 above)
Comment (yosriz) [reply sample/src/main/res/layout/wallet_activity.xml:63]: I really don't think that Sample app should serve as an example of the UI, and I really don't think it's that wrong to use a nested linear layouts in the sample case like this

**Pull request merged**